classes/fullstack/project1/client/src/components/LanguageSelector/LangTitleTest.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
```

Comment:
there is already a title component us ethe existing one

Reply:
You told me not to interact with existing components, I've created this one just to make sure it works, can be deleted of course